[2024-05-05T15:43:56.810+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-05-05T15:43:56.839+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: example_complex.delete_entry manual__2024-05-05T15:42:35.204677+00:00 [queued]>
[2024-05-05T15:43:56.850+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: example_complex.delete_entry manual__2024-05-05T15:42:35.204677+00:00 [queued]>
[2024-05-05T15:43:56.851+0000] {taskinstance.py:2303} INFO - Starting attempt 1 of 1
[2024-05-05T15:43:56.869+0000] {taskinstance.py:2327} INFO - Executing <Task(BashOperator): delete_entry> on 2024-05-05 15:42:35.204677+00:00
[2024-05-05T15:43:56.878+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=323) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-05-05T15:43:56.879+0000] {standard_task_runner.py:63} INFO - Started process 325 to run task
[2024-05-05T15:43:56.879+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'example_complex', 'delete_entry', 'manual__2024-05-05T15:42:35.204677+00:00', '--job-id', '39', '--raw', '--subdir', '/home/***/.local/lib/python3.12/site-packages/***/example_dags/example_complex.py', '--cfg-path', '/tmp/tmpir3g29y1']
[2024-05-05T15:43:56.881+0000] {standard_task_runner.py:91} INFO - Job 39: Subtask delete_entry
[2024-05-05T15:43:56.921+0000] {task_command.py:426} INFO - Running <TaskInstance: example_complex.delete_entry manual__2024-05-05T15:42:35.204677+00:00 [running]> on host edcd6c8e3dad
[2024-05-05T15:43:57.009+0000] {taskinstance.py:2644} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='example_complex' AIRFLOW_CTX_TASK_ID='delete_entry' AIRFLOW_CTX_EXECUTION_DATE='2024-05-05T15:42:35.204677+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-05-05T15:42:35.204677+00:00'
[2024-05-05T15:43:57.013+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-05-05T15:43:57.013+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2024-05-05T15:43:57.014+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'echo delete_entry']
[2024-05-05T15:43:57.033+0000] {subprocess.py:86} INFO - Output:
[2024-05-05T15:43:57.035+0000] {subprocess.py:93} INFO - delete_entry
[2024-05-05T15:43:57.036+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2024-05-05T15:43:57.037+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-05-05T15:43:57.083+0000] {taskinstance.py:1205} INFO - Marking task as SUCCESS. dag_id=example_complex, task_id=delete_entry, execution_date=20240505T154235, start_date=20240505T154356, end_date=20240505T154357
[2024-05-05T15:43:57.139+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-05-05T15:43:57.170+0000] {taskinstance.py:3482} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-05-05T15:43:57.181+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
